res/database/Default/user/speedminister.xml

```diff
@@ -21,10 +21,10 @@
         <person id="SpeedMinister-person-id019" first_name="Prof. Dr. Raimund" last_name="Holzner" creator="8605" created_by="speedminister" fictional="1" gender="1" country="D" />
         <person id="SpeedMinister-person-id027" first_name="Dirk" last_name="Kipling" creator="8605" created_by="speedminister" fictional="1" gender="1" country="D" />
 
-        <!-- TODO: Most non-fictional people in this file seem to be unused? -->
-        
-        <person id="SpeedMinister-person-id037" first_name="Den" last_name="Jakobson" first_name_original="Dennis" last_name_original="Jacobsen" creator="8605" created_by="speedminister" />
-        <person id="SpeedMinister-person-id038" first_name="Lianda" last_name="Mahoud" first_name_original="Randa" last_name_original="Chahoud" creator="8605" created_by="speedminister" />
+        <!-- TODO: Most non-fictional people in this file seem to be unused? Update: IMDB data added, TODO: move entries, TODO: add related programme, e.g. tt0978537: Ijon Tichy: Raumpilot -->
```

Comment:
Think SpeedMinister added them first and then created some programme for them.

Normally we do not really need the unused ones - because we have a person generator for a variety of countries. Yet of course the pre-generation of certain/specific persons might be interesting (because of hmm more "realistic _and_ common" names).

Reply:
I would prefer removing unused persons (not castable) from the database.

Reply:
hmm if someone makes the effort I'd rather prefer to add a TODO to add the related programme.

And in this case, I have found the missing programme. Speedminister had posted a newer version of his data: https://www.gamezworld.de/phpforum/viewtopic.php?pid=79070#p79070

I'll compare it and add it at a later date.